Cargo.toml

```diff
@@ -27,7 +27,7 @@ secp-recovery = ["secp256k1/recovery"]
 # Instead no-std enables additional features required for this crate to be usable without std.
 # As a result, both can be enabled without conflict.
 std = ["secp256k1/std", "bitcoin_hashes/std", "bech32/std"]
-no-std = ["hashbrown", "core2/alloc", "bitcoin_hashes/alloc", "secp256k1/alloc"]
+no-std = ["hashbrown", "core2/alloc", "bitcoin_hashes/alloc", "secp256k1/alloc", "serde/alloc"]
```

Comment:
This doesn't work because `serde` is optional but now `no-std` enables it.

Reply:
Oof, good catch. This is extremely misleading syntax on the part of the cargo folks. I stupidly read this as a conditional feature enabling, even though we've had endless discussions about that being impossible..